bin/pycbc_live

```diff
@@ -1169,13 +1175,19 @@ with ctx:
         results = {}
         evnt.live_detectors = set()
 
+        # Create objects to track whether the psd has been recalculated
```

Comment:
'objects' -> 'dict'

What does this mean exactly, is it that if the range doesn't change by much the psd doesn't get recalculated at each new time segment?

Reply:
Exactly, the arg `--psd-recalculate-difference` is used to determine if a new psd is too different from the current psd and the old one needs to be replaced ([link](https://github.com/gwastro/pycbc/blob/master/pycbc/strain/strain.py#L1681) to function).

We use the PSD to create the filter in the psd variation so need to keep track of when a new psd is being used by pycbc live. Will change it to say dict not just objects.